hilo.rb

```diff
@@ -4,4 +4,20 @@
 puts "You have 6 guesses."
 
 random_num = rand(1..100)
-puts "Random number: #{random_num}"
+
+# only allow integer input
+guess = gets
```

Comment:
Being pedantic, but from a users perpective you haven't requested that they guess yet.